extensions/blocks/podcast-player/templates/podcast-header.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Podcast Header template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+/**
+ * Block attributes
+ */
+$attributes = (array) $data['attributes'];
+
+/**
+ * Player data.
+ */
+
+$player_id                = (string) $data['playerId'];
+$title                    = (string) $data['title'];
+$cover                    = (string) $data['cover'];
+$link                     = (string) $data['link'];
+$track                    = ! empty( $data['tracks'] ) ? $data['tracks'][0] : array();
+$show_cover_art           = (bool) $attributes['showCoverArt'];
+$show_episode_description = (bool) $attributes['showEpisodeDescription'];
+?>
+
+
+<?php
+function render_podcast_title( $title, $link ) {
+	if ( ! isset( $title ) ) {
+		return;
+	}
+	?>
+
+	<?php if ( isset( $link ) ) : ?>
+		<a
+			class="jetpack-podcast-player__podcast-title"
+			href="<?php echo esc_url( $link ); ?>"
+			target="_blank"
+			rel="noopener noreferrer nofollow"
+		>
+			<?php echo esc_attr( $title ); ?>
+		</a>
+	<?php else : ?>
+		<span class="jetpack-podcast-player__podcast-title">
+			<?php echo esc_attr( $title ); ?>
+		</span>;
+	<?php endif; ?>
+<?php } ?>
+
+<?php
+function render_title( $player_id, $title, $link, $track ) {
+	if ( ! isset( $title ) && empty( $track ) && ! isset( $track['title'] ) ) {
+		return;
+	}
+	?>
+	<h2 id=<?php echo esc_attr( $player_id ); ?>__title" class="jetpack-podcast-player__title">
+		<?php if ( ! empty( $track ) && isset( $track['title'] ) ) : ?>
+			<span class="jetpack-podcast-player__current-track-title">
+				<?php echo $track['title']; ?>
```

Comment:
Missing escaping of output.
```suggestion
				<?php echo esc_html( $track['title'] ); ?>
```